src/Extensions/ContentReviewDefaultSettings.php

```diff
@@ -164,13 +165,51 @@ public function updateCMSFields(FieldList $fields)
                 TextField::create('ReviewFrom', _t(__CLASS__ . '.EMAILFROM', 'From email address'))
                     ->setDescription(_t(__CLASS__ . '.EMAILFROM_RIGHTTITLE', 'e.g: do-not-reply@site.com')),
                 TextField::create('ReviewSubject', _t(__CLASS__ . '.EMAILSUBJECT', 'Subject line')),
-                TextAreaField::create('ReviewBody', _t(__CLASS__ . '.EMAILTEMPLATE', 'Email template')),
+                $wysiwygConfig = HTMLEditorField::create(
+                    'ReviewBody',
+                    _t(__CLASS__ . '.EMAILTEMPLATE', 'Email template')
+                ),
                 LiteralField::create(
                     'TemplateHelp',
                     $this->owner->renderWith('SilverStripe\\ContentReview\\ContentReviewAdminHelp')
                 ),
             ]
         );
+        $wysiwygConfig->setEditorConfig($this->getTinyMCEConfig($wysiwygConfig->getEditorConfig()));
+    }
+
+    /**
+     * Get the TinyMCEConfig that should be used for the email template preview
+     *
+     * @return TinyMCEConfig
+     */
+    private function getTinyMCEConfig(
+        TinyMCEConfig $config
+    ): TinyMCEConfig {
+        $editorButtonsGroupSeparator = '|';
+        $allowedEditorButtons = [
+            'undo',
+            'redo',
+            $editorButtonsGroupSeparator,
+            'bold',
+            'italic',
+            'underline',
+            $editorButtonsGroupSeparator,
+            'bullist',
+            'numlist',
+            $editorButtonsGroupSeparator,
+            'sslink',
+            $editorButtonsGroupSeparator,
+            'formatselect',
+            $editorButtonsGroupSeparator,
+            'code',
+        ];
+
+        $config->setButtonsForLine(1, $allowedEditorButtons);
+        $config->setButtonsForLine(2, []);
+        $config->setButtonsForLine(3, []);
+
+        return $config;
```

Comment:
The config should be defined in `_config.php` and be given its own name - this allows developers to easily modify it in their own projects however they may so choose. Also there's no need to set a variable for `|`.

Reply:
@GuySartorelli removed the tinymce helper as we can put this in our projects as agreed doesn't need to live in this module.